src/python/library/tritonclient/utils/shared_memory/__init__.py

```diff
@@ -101,27 +58,52 @@ def create_shared_memory_region(triton_shm_name, shm_key, byte_size):
         The unique key of the shared memory object.
     byte_size : int
         The size in bytes of the shared memory region to be created.
+    create_only : bool
+        Whether a shared memory region must be created. If False and
+        a shared memory region of the same name exists, a handle to that
+        shared memory region will be returned and user must be aware that
+        the shared memory size can be different from the size requested.
```

Comment:
```suggestion
        the previously allocated shared memory size can be larger than the size requested.
```
Nit. Current language seems to imply the user can request a size that is larger than previously allocated. This was valid behavior before, but is not with your changes now. I think we want to make that clear here.